src/styles/text.tsx

```diff
@@ -11,3 +20,27 @@ export const H2 = styled.h2`
   font-size: 1.5rem;
   margin: 0;
 `;
+
+export const H4 = styled.h4`
+  color: ${COLORS.existingInputColor};
+  ${openSans.style}
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+`;
+
+export const H4Centered = styled(H4)`
```

Comment:
i think we can take out specific text styles like h4centered and keep more consistent general ones like just h4. instead, we can style children tags that live inside a container. for example inside the FormDiv you have, you can put this (H4 {
    text-align: center;
  }) which would achieve the same behavior